django/pyrog/migrations/0013_conditions_and_filters_sqlinput.py

```diff
@@ -0,0 +1,110 @@
+# Generated by Django 3.1.12 on 2021-09-09 09:28
```

Comment:
Not asking to change that but I'm wondering if this should have been 2 separate migrations?

Reply:
mhh I don't know, maybe..
yes probably
won't happen again